starter-code/iteration-1/main.ts

```diff
@@ -1,12 +1,31 @@
 // Iteration 1
 // 1. Create an array of strings
+let myArray : Array <string> = [];
 
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
-
+function addTask(task){
+myArray.push(task)
+console.log(`========== NEW TASK ==========
+    ${task}
+   `);
+   return myArray.length;
+}
 // 3. Create a function to list all tasks, it must show in the console de task.
+function listAllTasks(){
+ myArray.forEach(task=>{
+  console.log(task);
+ })
+}
 
 // 4. Create a function to delete a task, you must find the task inside the array and delete it.
+function deleteTask(task){
```

Comment:
Aquí también hacía falta que devolvieran algo, lo cual no está haciendo, aquí el ejemplo:
`function deleteTask(task):number{
let remove = myArray.indexOf(task)
myArray = myArray.splice(remove,1);

 console.log('====== TASK REMOVED ======');
 console.log(`Task "${task}" removed from the list`);
 return myArray.length; }`